CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/libkode/common")
 	message(FATAL_ERROR "Please do git submodule update --init --recursive")
 endif()
 
+option(ENABLE_TESTS "Enable unittests" OFF)
```

Comment:
All of KDE code uses BUILD_TESTING as the name of this option, I wouldn't mind if this project would use the same :)
No big deal otherwise.

Reply:
Done